core/src/main/java/org/arquillian/smart/testing/configuration/ObjectMapper.java

```diff
@@ -17,6 +17,11 @@
 class ObjectMapper {
 
     static <T extends ConfigurationSection> T mapToObject(Class<T> aClass, Map<String, Object> map) {
+        final ObjectMapper objectMapper = new ObjectMapper();
+        return objectMapper.readValue(aClass, map);
+    }
+
+    <T extends ConfigurationSection> T readValue(Class<T> aClass, Map<String, Object> map) {
```

Comment:
it's not necessary. We can remove this. but I created it, to keep it clean. & with `mapToObject` we have static factory method due to which we don't have to create `ObjectMapper` each time.

Reply:
> With mapToObject we have static factory method due to which you don't have to create ObjectMapper instance each time.

if it is a static method, then you don't create any instance of the class.
This approach is wrongly used in several cases in the ST code - instead of using classes such as `ConfigurationChecker` and `ConfigurationReader` where you need to create an instance of them and then call one method (without any need to keep the state) you can use them as static "util" classes.